### PR TITLE
Added the IAM Role create process

### PR DESCRIPTION
**Pull request by athiruma**
1. AWS IAM Role create process using console and terraform

### CODE REVIEW

File: iam/clouds/aws/payer_roles/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+7. Click on **Create Policy**. ( Policy will be created and listed on Policies )
+
+#### Create the IAM Role
+
+1. Click on **Roles**
+2. Click on **Create Role**
+3. Select the **Custom trust policy** from Trusted identity type.
+4. Paste the contents of *CloudGovernanceCostExplorerReadRole.json* file. \
+&nbsp;&nbsp;Note: Replace username with **IAM User** name, and AccountId with the **AWS AccountId**
+5Select the **CloudGovernanceCostExplorerReadPolicy** from the list of policies.
```
Comment (ebattat) [inline iam/clouds/aws/payer_roles/README.md:24]: 5Select - fix it

File: iam/clouds/aws/payer_roles/terrafom_create_role/main.tf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+output "role_name" {
+  value = aws_iam_role.cloud_governance_ce_read_role.name
+}
+
+resource "aws_iam_role" "cloud_governance_ce_read_role" {
+
+  name = "CloudGovernanceCostExplorerReadRole"
+
+  assume_role_policy = file("./../CloudGovernanceCostExplorerReadRole.json")
```
Comment (ebattat) [inline iam/clouds/aws/payer_roles/terrafom_create_role/main.tf:18]: only 1 policy ?

Comment (athiruma) [reply iam/clouds/aws/payer_roles/terrafom_create_role/main.tf:18]: yes.

File: iam/clouds/aws/payer_roles/README.md (hunk 1 above)
Comment (athiruma) [reply iam/clouds/aws/payer_roles/README.md:24]: ok

**Pull request merged**